accessibility-checker-engine/src/v2/checker/accessibility/rules/rpt-input-rules.ts

```diff
@@ -163,6 +163,8 @@ let a11yRulesInput: Rule[] = [
         run: (context: RuleContext, options?: {}): RuleResult | RuleResult[] => {
             // See https://www.w3.org/WAI/WCAG21/Techniques/failures/F65
             const ruleContext = context["dom"].node as Element;
+            //skip the rule
+            if (RPTUtil.isNodeHidden(ruleContext)) return null;
```

Comment:
Not sure I understand the justification here. Inputs are interactive and shouldn't be hidden. Maybe if it also has a tabindex=-1 or is disabled?

Reply:
removed, this is probably better to handle case by case